azure-pipelines.yml

```diff
@@ -42,27 +42,41 @@ jobs:
           BuildType: Debug
 
       # clang-5 and newer compilers
-      Ubuntu Bionic clang-13 Release:
+      Ubuntu Bionic clang-14 Release:
         imageName: 'ubuntu-18.04'
         Compiler: clang
-        CompilerVer: 13
+        CompilerVer: 14
         BuildType: RelWithDebInfo
 
       ${{ if ne(variables['Build.Reason'], 'IndividualCI') }}:
-        Ubuntu Bionic clang-13 Release INVERTED=ON:
+        Ubuntu Bionic clang-14 Release INVERTED=ON:
           imageName: 'ubuntu-18.04'
           Compiler: clang
-          CompilerVer: 13
+          CompilerVer: 14
           BuildType: RelWithDebInfo
           Inverted: ON
 
-      Ubuntu Bionic clang-13 Debug INVERTED=ON:
+      ${{ if ne(variables['Build.Reason'], 'IndividualCI') }}:
+        Ubuntu Bionic clang-14 Debug:
+          imageName: 'ubuntu-18.04'
+          Compiler: clang
+          CompilerVer: 14
+          BuildType: Debug
+
+      Ubuntu Bionic clang-14 Debug INVERTED=ON:
         imageName: 'ubuntu-18.04'
         Compiler: clang
-        CompilerVer: 13
+        CompilerVer: 14
         BuildType: Debug
         Inverted: ON
 
+      ${{ if ne(variables['Build.Reason'], 'IndividualCI') }}:
+        Ubuntu Bionic clang-13 Release:
+          imageName: 'ubuntu-18.04'
+          Compiler: clang
+          CompilerVer: 13
+          BuildType: RelWithDebInfo
+
       ${{ if ne(variables['Build.Reason'], 'IndividualCI') }}:
```

Comment:
Please remove this line to turn on `Ubuntu Bionic clang-13 Debug` for all cases.
In another case for `IndividualCI` we will have only:
-Ubuntu Bionic clang-14 Release
-Ubuntu Bionic clang-14 Debug INVERTED=ON
-Ubuntu Bionic clang-11 Debug
...